praw/models/reddit/collections.py

```diff
@@ -165,6 +165,27 @@ def update_description(self, description: str):
             data={"collection_id": self.collection_id, "description": description},
         )
 
+    def update_display_layout(self, display_layout: str):
+        """Update the collection's display layout.
+
+        :param display_layout: Either `TIMELINE` or `GALLERY`.
+
+        Example usage:
+
+        .. code-block:: python
+
+            collection = reddit.subreddit("SUBREDDIT").collections("some_uuid")
+            collection.mod.update_display_layout("GALLERY")
+
+        """
+        self._reddit.post(
```

Comment:
Does this return anything?

Reply:
No....why do you ask? 🤔

Reply:
If it returned anything we could utilize it if is useful.

Reply:
Well, this function is similar to update_title and update_description, so I don't think it should return anything. Because you can access a collection's layout with properties.